inspirehep/ext/jinja_filters/record.py

```diff
@@ -396,5 +396,24 @@ def citation_phrase(count):
 
     @app.template_filter()
     def number_of_search_results(query, collection_name):
+        """
+        Filter used to show total number of results out of filtered ones.
+        """
+        from flask import session
+        import datetime
```

Comment:
Minor thing, these import should be safe to keep at module level to avoid hitting these lines every time the filter is used (even if they will quickly return as it is already imported, but yeah).